src/Extensions/Abstractions/EventSourcesEventIds.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions
```

Comment:
Why is this in the Extensions namespace?

Reply:
All those projects called extensions since it's an implementation or extension of interfaces created in https://github.com/dotnet/extensions repository.
This enum created to keep track of events that we creating, logging is only one application of them, we also have it in TimedScopes and Hosting.Services (fallback in case of logging initialization failure).
Moving this into Logger will force dependency of TimedScopes on the Logger that I would like to avoid. 
An interesting and probably right thing to do here would be to extract all event sources into the separate package because it's something specific to our implementation.

Reply:
I've spent some time trying out the idea of extracting all event sources into separate NuGet package, but it will have the following consequences:
- TimedScopes, Logger, Service would have to have empty implementations in order for these packages to be independent.
- Service EventSource would have to be public since it's used before DI
- The most important part is that in your service creation code user would have to remember to add a special line that will register event sources and if it's skipped log information won't be sent at all.

Taking all of it in considerations and the fact that currently all of event source implementations marked as `internal` (only EvenIds exposed `public`), registered in DI and easily replaceable with other implementations I think it's better to keep it like this for now.

I should be quite easy to extract it in the future if needed.